services/src/handlers/tasks.rs

```diff
@@ -0,0 +1,271 @@
+use crate::error::Result;
+use crate::tasks::{TaskDb, TaskListOptions};
+use crate::{contexts::Context, tasks::TaskId};
+use actix_web::{web, FromRequest, Responder};
+use serde::{Deserialize, Serialize};
+
+pub(crate) fn init_task_routes<C>(cfg: &mut web::ServiceConfig)
+where
+    C: Context,
+    C::Session: FromRequest,
+{
+    cfg.service(
+        web::scope("/tasks")
+            .service(web::resource("/list").route(web::get().to(list_handler::<C>)))
+            .service(web::resource("/{task_id}/status").route(web::get().to(status_handler::<C>))),
```

Comment:
the REST url should be singular task: `/task/id/status`

Reply:
But it is our endpoint for tasks. Within, you have one taskId… right?
What is the guideline, maybe you can send me something, for this issue and my interest :o.

Reply:
I think you are right
https://restfulapi.net/resource-naming/
https://cloud.google.com/apis/design/resource_names

I guess I have to rethink our whole API :D

Reply:
:muscle: